actions/lib/actions.py

```diff
@@ -93,7 +93,13 @@ def get_node(self, node):
             FROM Cirrus.Nodes
             WHERE CoreNodeID=@CoreNodeID"""
             kargs = {'CoreNodeID': orion_node.npm_id}
-            data_ncm = self.query(swql_ncm, **kargs)
+            try:
+                data_ncm = self.query(swql_ncm, **kargs)
+            except:
+                # Create an empty dict to allow remaining code to fail gracefully
+                data_ncm = {}
+                msg = "Connection to NCM failed. NCM not installed"
+                self.logger.info(msg)
```

Comment:
This would be cleaner as a single line:

```suggestion
                self.logger.info("Connection to NCM failed. NCM not installed")
```

Reply:
Agreed. Changed message logic to single line